example/App/UI/Element.purs

```diff
@@ -160,9 +160,9 @@ formlessField
    . IsSymbol sym
   => ToText e
   => Newtype (form Record F.FormField) (Record fields)
-  => Newtype (form Variant F.InputField) (Variant inputs)
+  => Newtype (form Variant F.InputFunction) (Variant inputs)
```

Comment:
@dariooddenino Helper classes that previously used the `InputField` variant will need to be updated to use the `InputFunction` variant instead, the only class on the component. The variant was changed to support `InputFunction (input -> input)` and modification.